src/styles/styles.scss

```diff
@@ -0,0 +1,158 @@
+.body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+h2 {
+  margin: 0;
+  padding: 0;
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+div {
+  margin: 0;
+  padding: 0;
+}
+
+button {
+  margin: 0;
+  padding: 0;
```

Comment:
the same with button

Reply:
If I remove the padding reset, my styles won't work